### PR TITLE
[iOS] Commit snapshots with CI

### PR DESCRIPTION
**Pull request by ry-itto**
## Issue
- close #ISSUE_NUMBER

## Overview (Required)
- Commit snapshots on pushed to main branch (PR merged)
  - Commit only `ios/**` changed.
- Added because there was a problem running in the local environment
  - https://github.com/DroidKaigi/conference-app-2021/pull/528#issuecomment-873368433

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by ry-itto** [comment]
!github take-ios-snapshots

**Comment by ry-itto** [comment]
!github take-ios-snapshots

**Comment by ry-itto** [comment]
!github take-ios-snapshots

File: ios/Tests/AboutFeatureTests/AboutFeatureTests.swift [hunk 1]
```diff
@@ -3,6 +3,10 @@ import TestUtils
 import XCTest
 
 final class AboutFeatureTests: XCTestCase {
+    override func setUp() {
```
Comment (ostk0069) [inline ios/Tests/AboutFeatureTests/AboutFeatureTests.swift:6]: It seems better to create custom XCTestCase class to make sure we don't have to add this everytime

**Comment by ostk0069** [comment]
Please update the current README make sure contributors won't update the snapshots by themselves.

File: ios/Tests/AboutFeatureTests/AboutFeatureTests.swift (hunk 1 above)
Comment (ry-itto) [reply ios/Tests/AboutFeatureTests/AboutFeatureTests.swift:6]: I don't think either will change.
When using custom XCTestCase, it is necessary to inherit so as not to forget.

**Comment by ry-itto** [comment]
@ostk0069 
I fixed 🙏🏼 

> Maybe adding document comment in PreviewSnapshot.swift as well is way better
I think document comment in implementation file should be about implementation only not included operational comments.

**Comment by ostk0069** [comment]
> I think document comment in implementation file should be about implementation only not included operational comments.

What I meant was adding some comment what contributors could stop creating snapshot  themselves. It does not matter it is as a document comment or just a simple comment.

**Review by ostk0069 — APPROVED** [review]

**Pull request merged**